libs/localization/assets/locales/en/char_TravelerGeo.json

```diff
@@ -4,8 +4,5 @@
   },
   "c2": {
     "key": "Rockcore Meltdown DMG"
-  },
-  "c4": {
-    "energyRestore": "Energy restored per shockwave hit"
```

Comment:
Why are we removing this to use a more vague text?

Reply:
Other characters also use simply "Energy Regeneration". Venti's Burst, for example, doesn't say "Energy restored upon Burst ending", it just uses the standardized string. That's the point of having Sheet.

Reply:
I am all for standardization, but not at the cost of context. I think its good to prompt user on how/why a condition occurs, (and use a manually translated string) than to create a generalized standardized string that gives less information. Especially since in the case of geo traveler, this string has already been translated to several languages already, I see this as a net loss to use a generalized string.

Reply:
that's valid, ig. same for mona, y'reckon?

Reply:
The Mona string is fine.